components/Chips/src/ColorThemer/MDCChipViewColorThemer.h

```diff
@@ -1,72 +0,0 @@
-// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#import "MaterialChips.h"
-#import "MaterialColorScheme.h"
-
-#import <Foundation/Foundation.h>
-
-/**
- The Material Design color system's themer for instances of MDCChipView.
-
- @warning This API will eventually be deprecated. See the individual method documentation for
- details on replacement APIs.
- Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
- */
-@interface MDCChipViewColorThemer : NSObject
-
-@end
-
-@interface MDCChipViewColorThemer (ToBeDeprecated)
```

Comment:
Please do not delete this class (or any other class) until it's been explicitly marked deprecated for one release version.

Reply:
Good catch. These are marked deprecated in 94.5.0, whose release is currently in progress.